src/filtrering/filtrering-filter/filterform/hendelser-filterform.tsx

```diff
@@ -35,8 +34,12 @@ export function HendelserFilterform({
     };
 
     useEffect(() => {
-        setHendelserValg(filtervalg[form]);
-    }, [filtervalg, form]);
+        if (filtervalg[form]) {
+            setHendelserValg(filtervalg[form][0]);
+        } else {
+            setHendelserValg('');
+        }
+    }, [filtervalg, hendelserValg, form]);
```

Comment:
Klarte ikke helt å se hvorfor disse endringene var nødvendig, men hvis det er testet så er det ok for min del 👍

Reply:
Emil hjalp meg med dette, vi hadde problemer med at hendelserValg var blitt en array og ikke en string slik det var før.